screen-orientation/lock-basic.html

```diff
@@ -27,17 +26,27 @@
     'landscape-secondary',
     'portrait-primary',
   ];
-  const promises = orientations.map(type => {
-    const returnType = screen.orientation.lock(type);
+  for (const orientation of orientations) {
+    const returnType = screen.orientation.lock(orientation);
     assert_true(returnType instanceof Promise);
-  });
-  return Promise.all(promises).finally(() => {
-    screen.orientation.unlock();
-  });
+    await returnType;
```

Comment:
I feel like we should assert there result here:

```JS
const promiseToChange = screen.orientation.lock(orientation);
assert_true(promiseToChange instanceof Promise, "Expected an instance of Promise");
const resultingOrientation = await promiseToChange;
assert_equals(resultingOrientation, orientation, "Expected orientation to change");
```

Reply:
OK, I suppose you meant checking orientation type in last assertion.